vars/managerPipeline.groovy

```diff
@@ -343,7 +363,51 @@ def call(Map pipelineParams) {
                     }
                 }
             }
+            success {
```

Comment:
I think it is better to put this in a stage of its own (I don't know the general context, so maybe it is not what you usually do).
Any stage runs only if prior stages ended successfully.
You can put:
when {
	expression { some condition }
}
so if the condition is not st, the step will be shown as skipped on the stage table. This is nice to see.
When if failed - you see what failed. As it is written here, you may see that the job is success, but still "red". You can't mark a failures when you are already on success stage.

Reply:
I wanted to create a stage for it, but I got an error when I tried to set one under the `post` header (I guess it's not allowed), but creating a different stage for it is a good Idea.

But the thing is I don't want the triggering job to fail when the downstream job doesn't exist. That's why I specifically changed the `JobEnabled` function

Reply:
agree with @hagitsegev , wrap it with try .. catch and use currentBuild.currentResult to get job status you need 'SUCCESS'